Gemfile

```diff
@@ -13,7 +13,7 @@ group :development do
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'rake', '10.4.2'
-  gem 'rubocop'
+  gem 'rubocop', '~> 0.43.0'
```

Comment:
Hmm, I agree, rubocop version is somehow involved. I just don't understand. So looking at CU it indeed uses _0.43.0_ which is expected, since it is the latest release.

However, if I clean my gemset and execute a _bundle install_, I also should get this latest version, even w/o your change, right? But I get 0.41.2!?